app/models/agents/http_status_agent.rb

```diff
@@ -12,6 +12,7 @@ class HttpStatusAgent < Agent
 
     form_configurable :url
     form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
+    form_configurable :changes_only, type: :array, values: ['true', 'false']
```

Comment:
Oh I did not notice the line above, `:boolean` uses radio buttons instead of a dropdown. I think the code should just work if you switch to it as it also should send `true` or `false` as a string. If it would require code changes its fine to keep the dropdowns.